docs/uml-class-diagrams/middleware01/production/MiddlewareGisManagerImplementation/MiddlewareGisManagerImplementation.puml

```diff
@@ -2,11 +2,15 @@
 
 package edu.oakland.production.middleware01 {
     class MiddlewareGisManagerImplementation {
+        - MiddlewareGisProcessor middlewareGisProcessor
         __
+        + MiddlewareGisManagerImplementation(MiddlewareGisProcessor middlewareGisProcessor)
```

Comment:
This references database, right? If so, should it take a `DatabaseGisInterface` in the contructor?
```suggestion
        - DatabaseGisInterface databaseGisInterface
        __
        + MiddlewareGisManagerImplementation(DatabaseGisInterface databaseGisInterface)
```

Reply:
Very good point, thanks for catching that!